adoc/MAIN.release-notes.adoc

```diff
@@ -9,13 +9,29 @@ ifeval::['{release_type}' != 'public']
 !!! This is a preview release. Not to be distributed outside of SUSE !!!
 endif::[]
 
+SUSE CaaS Platform is an enterprise-ready Kubernetes-based container management solution
+
+These release notes are updated periodically. The latest version is always available at https://www.suse.com/releasenotes. General documentation can be found at: https://www.suse.com/documentation/suse.caasp-4.
```

Comment:
The `https://www.suse.com/documentation/suse.caasp-4` URL is definitely wrong.

* On the old site, it'd be `https://www.suse.com/documentation/suse-caasp-4`. (But we most likely won't be able to publish there in September because the TSA will run out.)
* On the new site, it'll be `https://documentation.suse.com/suse-caasp/4/`. (That one should really be live in September but is completely dead currently, so will create minor pains in the coming ~four days.)

Reply:
I went for ` https://documentation.suse.com/suse-caasp/4/ `, once this version of the RN is out, the new site should be up as well.